package.json

```diff
@@ -71,6 +71,7 @@
     "throat": "^4.1.0",
     "tiny-attribution-generator": "0.1.2",
     "tmp": "0.0.33",
+    "v8": "0.1.0",
```

Comment:
Could you explain the reasoning behind introducing a library (9-year-old) instead of using the `v8` module? The functions needed appear to be available in Node.js 18 LTS. Please refer to the documentation:

- [v8.getHeapSpaceStatistics](https://nodejs.org/docs/latest-v18.x/api/v8.html#v8getheapspacestatistics)
- [v8.getHeapStatistics](https://nodejs.org/docs/latest-v18.x/api/v8.html#v8getheapstatistics)

Reply:
As discussed in the sync call, I removed this npm package in favor of the built-in `v8` module